tests/remote/filter_predicate_aliases.tftest.hcl

```diff
@@ -0,0 +1,410 @@
+variables {
+  name            = "TF Test Filter with aliases predicate"
+  predicate_value = "fancy"
+  aliases_predicates = setproduct(
+    ["aliases"],
+    concat([
+      "contains",
```

Comment:
Why do we have vars for `predicate_types_equals` and `predicate_types_exists` but not for these ones?

Reply:
I just never came back around to making them but yeah these can be added